#664 Add @OneToMany annotation as alternative to @BelongsTo (create-table statements for other DBs)

cschabl opened the pull request.

ipolevoy commented:
@cschabl I'm scratching my head. This commit was merged to master, but the code is missing there. Can you take a quick look and see why? Appreciate help!

cschabl commented:
@ipolevoy , I checked the changes on production code. And they are on master:
[Registry.java](https://github.com/javalite/activejdbc/blob/master/activejdbc/src/main/java/org/javalite/activejdbc/Registry.java), lines 257-258, 363-367, 378-395
New class [HasMany.java](https://github.com/javalite/activejdbc/blob/master/activejdbc/src/main/java/org/javalite/activejdbc/annotations/HasMany.java)

ipolevoy commented:
@cschabl the problem is that some lines are gone from the SQL files. I tracked this to merging of your pull request: https://github.com/javalite/activejdbc/pull/685

For instance, the tables in a file `mysql_schema.sql`, tables `teams` and `players` were replaced by 

* bands
* genres
* musicians

not sure why they would be just replaced? The builds are broken because `teams` and `players` are missing. 

![selection_001091](https://user-images.githubusercontent.com/500941/40000899-a773614a-5752-11e8-9cd7-5312d2272b91.png)

cschabl commented:
@ipolevoy, this is due to a wrongly resolved merge conflict between PR #685 and PR #696 or sort of "lost update". After you have merged my PR #670 into master you realized that I've only updated h2_schema.sql for the tables teams and players and told me to update the other schema.sql files as well. I have done this in the PR #696. In the meantime I've implemented the feature "Make Many2Many annotation repeatable" (PR #685) in which I introduced the tables bands, genres and musicians. Therefore, I had to add create-table-statements for these tables to all schema.sql files.

cschabl commented:
@ipolevoy, I created a second branch with the missing create-table-statements you can pull (#718)

ipolevoy commented:
@cschabl thank you for unearthing this, and for the new PL - I hope it works.
